clang/test/SemaCUDA/call-host-fn-from-device.cu

```diff
@@ -12,7 +12,7 @@ extern "C" void host_fn() {}
 struct Dummy {};
 
 struct S {
-  S() {}
+  S() { x = 1; }
```

Comment:
Can we make the purpose of non-trivial constructor more descriptive, here and in other places?
E.g. `S() { static int nontrivial_ctor = 1; }`

Reply:
will do